### ISSUE TITLE
Pose to Image, Segmentation Mask Question

### ISSUE THREAD

**Issue by stanleyshly** [CompVis/taming-transformers#28]
I was hoping to reimplement the pose to image portion of the paper with a couple modifications, does anyone have any information for the range of possible values allowed in segmentation, and also what color they correspond to?
Also, can you share the config and how you trained this model?

**Comment by rromb** [comment]
Hi. We represent the keypoints as images, on which we trained a VQGAN to obtain a tokenized discrete sequence. When training the transformer in the conditional pose-to-image setting, we used this latent representation as the conditioning sequence. Note that the transformer is trained in the space of another VQGAN which is pretrained on DeepFashion person images (very similar to the other experiments).

**Comment by stanleyshly** [comment]
I may be getting this wrong, but you first train only VQGAN to basically be an autoencoder, then add the transformer and target output later?

**Comment by rromb** [comment]
Yes. The VQGAN does not need to know anything about the downstream conditional task. It can also be trained on a different dataset and transferred to another when the transformer is trained during the second stage.

**Comment by stanleyshly** [comment]
Okay I think I almost got it figured out but I'm still a bit confused on dimensions. Do I train the VQGAN on the 256x256 samples, then run the 16x16 transformer on that?
Or do I train the the VQGAN 256x256, then use the dynamic graph model to encode any sized image, and only finally run the 16x16 sliding transformer on the encoded dense representation of the image?
